data/formations_basic.txt

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2019-2021 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+
+# Formation where ships form a line behind the formation leader.
+formation "Trailing line"
+	flippable y
+	position 0 200
+	position 0 400
+	position 0 600
+	position 0 800
+	position 0 1000
+	position 0 1200
+	position 0 1400
+	position 0 1800
+	position 0 2000
+	position 0 2200
+	position 0 2400
+	position 0 2600
+	position 0 2800
+	position 0 3000
+
```

Comment:
I assume you plan on adding more positions later. Currently, the escorts that do not have positions just cluster under the flag ship.

Reply:
Yes, adding more positions would allow for larger formations. My original plan (in PR #4471) was to automatically generate the positions:
https://github.com/petervdmeer/endless-sky/blob/d02bf3bf596716b780b2c1038761b917a1e8915b/data/formations.txt#L432

But the automatic generation of positions didn't pass the review, so manually adding all positions is the way to go now.